### DISCUSSION TITLE
How can I make the auto-complete of noice fill with first tab?

### DISCUSSION THREAD

**Discussion by danielo515** [LazyVim/LazyVim#2230] [Q&A]
I like the deafault neovim autocompletion of the cmd, as soon as I hit tab, it goes to the most possible auto completable item. However, noice auto-completes to the most common term, then you need to tab again to cycle through the options. Is it possible to auto-complete and jump to the first option just with the first tab?

**Comment by dpetka2001** [comment]
Use `set wildmode=full`. See `:h 'wildmode'` for more information. For permanent setting you should set it in your `options.lua` file like so `vim.opt.wildmode = "full"`.

**Comment by danielo515** [comment]
That did the trick, thanks

On Wed, Dec 20, 2023 at 5:53 PM Iordanis Petkakis ***@***.***>
wrote:

> Use set wildmode=list:full. See :h 'wildmode' for more information. For
> permanent setting you should set it in your options.lua file.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LazyVim/LazyVim/discussions/2230#discussioncomment-7910423>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AARKJWLBCJ25YVMN7L5DA4LYKMJY3AVCNFSM6AAAAABA5CKZAKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TSMJQGQZDG>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
---
https://danielorodriguez.com
